hc/api/tests/test_sendalerts.py

```diff
@@ -25,8 +25,10 @@ def test_it_handles_few(self, mock):
 
         handled_names = []
         for args, kwargs in mock.call_args_list:
-            handled_names.append(args[0].name)    
-        self.assertEqual(names, handled_names)
+            handled_names.append(args[0].name)
+        handled_names.reverse()    
```

Comment:
We resolved to leave this test as is, right? @muthash Remove the reference to reverse the handled_names list.

Reply:
Ignore this comment, Leave the test as is in your branch 😀